File: Domain/Others/Lecturer.cs
namespace Domain.Others
{
    public class Lecturer
    {
        public Guid Id { get; set; } = new Guid();
        public string FullName { get; set; }
        public string Prefix { get; set; }
        public string Position { get; set; }
        public string Image { get; set; } = "user-default.jpg";
        public string Expert { get; set; }
        public bool Hidden { get; set; } = false;
        public DateTime CreatedAt { get; set; } = new DateTime();
    }
}

File: client-app/src/models/Lecturer.ts
import { RcFile } from "antd/es/upload";

export interface Lecturer {
  id: string;
  fullName: string;
  position: string;
  image?: string | null;
  prefix: string;
  expert?: string;
  hidden: boolean;

  imagePreview?: string;
}

export interface LecturerCreate {
  fullName: string;
  position: string;
  prefix: string;
  expert?: string;
  
  fileImage: RcFile;
}

export interface LecturerUpdate extends LecturerCreate {
  id: string;
}
